site-service/src/main/java/uk/ac/ox/ndph/mts/site_service/SiteServiceApplication.java

```diff
@@ -18,7 +18,9 @@
      *
      */
     public static void main(String[] args) {
-        LOGGER.info(Application.STARTUP.message());
+        if (LOGGER.isInfoEnabled()) {
```

Comment:
No need for this log level can be set from application properties

Reply:
Fixed as required